packages/excalidraw/fonts/index.ts

```diff
@@ -44,10 +48,19 @@ export class Fonts {
       >
     | undefined;
 
+  private static _initialized: boolean = false;
+
   public static get registered() {
+    // lazy load the font registration
     if (!Fonts._registered) {
-      // lazy load the fonts
       Fonts._registered = Fonts.init();
+    } else if (!Fonts._initialized) {
+      // case when host app register fonts before they are lazy loaded
+      // don't override whatever has been previously registered
+      Fonts._registered = new Map([
+        ...Fonts.init().entries(),
+        ...Fonts._registered.entries(),
```

Comment:
is this in anticipation for registering custom fonts in runtime? Otherwise `Fonts.init().entries()` already gives you all the fonts anyway

Reply:
yup, preparation for custom fonts